spec/features/feedex_organisation_summary_spec.rb

```diff
@@ -0,0 +1,69 @@
+require 'rails_helper'
+require 'gds_api/test_helpers/support_api'
+
+feature "Summary of Organisation feedback" do
+  include GdsApi::TestHelpers::SupportApi
+  background do
+    login_as create(:user)
+  end
+
+  before do
+    stub_summary_sorted_by('last_7_days')
+    explore_anonymous_feedback_with(organisation: "Cabinet Office")
+  end
+
+  scenario "defaults to sorting feedback by last 7 days" do
+    expect(page).to have_content("Feedback for Cabinet Office")
+    expect(organisation_summary_results).to eq(organisation_summary)
+    expect(page).to have_selector('th.sorted-column', text: "7 days")
+  end
+
+  scenario "organisation feedback table can be sorted by path, 7, 30 and 90 days" do
```

Comment:
You can move the loop to outside the scenario, and have multiple scenarios auto-generated:
```ruby
{
  path: 'Page',
  last_7_days: '7 days',
  last_30_days: '30 days',
  last_90_days: '90 
}.each do |param, name|
  scenario "organisation feedback table can be sorted by #{name}" do
    stub_summary_sorted_by(param.to_s)
    within '.table-sortable thead' do
      click_on name
    end
    expect(organisation_summary_results).to eq(organisation_summary)
    expect(page).to have_selector('th.sorted-column', text: name)
    expect(page).to have_no_selector('th a', text: name)
  end
end
```